server/channels/app/user.go

```diff
@@ -658,6 +658,30 @@ func (a *App) GetUsersNotInChannelPage(teamID string, channelID string, groupCon
 	return a.sanitizeProfiles(users, asAdmin), nil
 }
 
+func (a *App) GetUsersNotInAbacChannel(teamID string, channelID string, groupConstrained bool, cursorID string, limit int, asAdmin bool, viewRestrictions *model.ViewUsersRestrictions) ([]*model.User, *model.AppError) {
```

Comment:
We can have ctx as function parameter here, and you won't need to create a new context. We already have it ready in api layer.